src/base/QXmppMixIq.cpp

```diff
@@ -75,6 +198,8 @@ class QXmppMixIqPrivate : public QSharedData
 ///
 /// The client subscribes to MIX nodes or unsubscribes from MIX nodes.
 ///
+/// \deprecated This is deprecated since QXmpp 1.7. Use QXmppMixSubscriptionUpdate instead.
```

Comment:
should actually better be `Use QXmppMixManager for this functionality` since the `QXmppMixSubscriptionUpdateIq` is private.